src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -523,13 +523,22 @@ const NetworkModificationNodeEditor = () => {
                         if (data[0][key] === null) {
                             delete data[0][key];
                         }
+                        for (var i = 0; i < copiedModifications.length; i++) {
```

Comment:
No need to roll all the loop
```
if (copiedModifications.some((m) => m === modificationUuid)) {
  setCopiedModifications([]);
}
```

Reply:
includes is even shorter because it's miss named, should be renamed `copiedModificationUuids` maybe ?